### ISSUE TITLE
Implement unicode dinput interface

### ISSUE THREAD

**Issue by HybridEidolon** [HybridEidolon/psobbaddonplugin#16]
It turns out some systems initialize dinput in unicode mode and it's not a PSOBB thing.

Currently, the addon just kills the game.

**Comment by ghost** [comment]
Yeah, this would be great. I have this issue and I can't use any plugins included here because of it.
